File: specification/support/resource-manager/Microsoft.Support/stable/2020-04-01/examples-js/GetProblemClassification.js
const { MicrosoftSupport } = require("@azure/arm-support");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Get problem classification details for a specific Azure service.
 *
 * @summary Get problem classification details for a specific Azure service.
 * x-ms-original-file: specification/support/resource-manager/Microsoft.Support/stable/2020-04-01/examples/GetProblemClassification.json
 */
async function getsDetailsOfProblemClassificationForAzureService() {
  const subscriptionId = "00000000-0000-0000-0000-000000000000";
  const serviceName = "service_guid";
  const problemClassificationName = "problemClassification_guid";
  const credential = new DefaultAzureCredential();
  const client = new MicrosoftSupport(credential, subscriptionId);
  const result = await client.problemClassifications.get(serviceName, problemClassificationName);
  console.log(result);
}

getsDetailsOfProblemClassificationForAzureService().catch(console.error);

File: specification/support/resource-manager/Microsoft.Support/stable/2020-04-01/examples-java/GetProblemClassification.java
/** Samples for ProblemClassifications Get. */
public final class Main {
    /*
     * x-ms-original-file: specification/support/resource-manager/Microsoft.Support/stable/2020-04-01/examples/GetProblemClassification.json
     */
    /**
     * Sample code: Gets details of problemClassification for Azure service.
     *
     * @param manager Entry point to SupportManager.
     */
    public static void getsDetailsOfProblemClassificationForAzureService(
        com.azure.resourcemanager.support.SupportManager manager) {
        manager
            .problemClassifications()
            .getWithResponse("service_guid", "problemClassification_guid", com.azure.core.util.Context.NONE);
    }
}
